crates/runtime/src/http/v1/chat.rs

```diff
@@ -221,3 +226,29 @@ impl OpenaiErrorEvent {
         }
     }
 }
+
+#[must_use]
+pub fn openai_error_to_response(e: OpenAIError) -> Response {
+    match e {
+        OpenAIError::InvalidArgument(_) => (StatusCode::BAD_REQUEST, e.to_string()).into_response(),
+        OpenAIError::ApiError(api_error) => {
+            let error_response = serde_json::json!({
```

Comment:
Why doesn't `error_response` doesn't naturally serialise as this `json!` does

Reply:
It can be serialized, but the type will be `Json<ApiError>` and there is no `into_response` implementation for this.
So have to construct json value directly.